plugins/mumble_plugin_linux.h

```diff
@@ -23,7 +19,8 @@
 #include "mumble_plugin.h"
 
 pid_t pPid;
-static PTR_TYPE_CONCRETE pModule;
+int is64Bit;
```

Comment:
Named like a bool. Should be a bool.

Reply:
Yes, it should, but C doesn't have a boolean data type, as pointed out by @hacst.